app/code/Magento/QuoteGraphQl/Model/Resolver/UpdateCartItems.php

```diff
@@ -14,6 +14,7 @@
 use Magento\Framework\GraphQl\Exception\GraphQlNoSuchEntityException;
 use Magento\Framework\GraphQl\Query\ResolverInterface;
 use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GiftMessage\Api\ItemRepositoryInterface;
```

Comment:
I see that we got a pretty "fat" resolver and looks like it will be even bigger over time. 
I would recommend performing some refactoring here. The idea is to create a separate class and put the `processCartItems` and `updateGiftMessageForItem` methods there. 

In that way, we will reduce the number of dependencies of the resolver as well as the overall complexity.

What do you think?

Reply:
**Done** I moved these methods to separate class
Thanks